rest-api-spec/src/main/resources/rest-api-spec/test/search/220_no_types.yml

```diff
@@ -0,0 +1,46 @@
+---
+"No type returned":
```

Comment:
To check I'm following -- this is our approach for the testing `search` API (as opposed to duplicating the REST tests, as we do for other APIs)? This test also got me thinking that we should probably deprecate/ disallow `type` queries.

Reply:
Does this mean that for _search, `include_type_name=false` means that `_type` should not be returned as part of the hits? Does the flag do anything else for _search?

Reply:
@javanna No, this is all it does indeed.

Reply:
Thanks for the explanation.